routes/boxHistory.js

```diff
@@ -52,6 +56,60 @@ boxHistoryRouter.put('/update', async (req, res) => {
   }
 });
 
+// Adds a box to the Box History table
+boxRouter.post('/', async (req, res) => {
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'boxRouter' is not defined.